CLI.md

```diff
@@ -958,15 +965,39 @@ optional arguments:
 $ mxpy wallet derive --help
 usage: mxpy wallet derive [-h] ...
 
-Derive a PEM file from a mnemonic or generate a new PEM file (for tests only!)
+DEPRECATED COMMAND, replaced by 'wallet convert'
 
 positional arguments:
   pem            path of the output PEM file
 
 optional arguments:
   -h, --help     show this help message and exit
   --mnemonic     whether to derive from an existing mnemonic
-  --index INDEX  the account index
+  --index INDEX  the address index
+
+```
+### Wallet.Convert
+
+
+```
+$ mxpy wallet convert --help
+usage: mxpy wallet convert [-h] ...
+
+Convert a wallet from one format to another
+
+optional arguments:
+  -h, --help                                      show this help message and exit
+  --infile INFILE                                 path to the input file
+  --outfile OUTFILE                               path to the output file
+  --in-format {raw-mnemonic,keystore-mnemonic,keystore-secret-key,pem}
```

Comment:
nitpicking, but I think that the input format can be programmatically figured out (might be a new flag `--detect-in-format`)